src/app/components/dotted-menu/movie-card-menu.component.spec.ts

```diff
@@ -16,10 +17,27 @@ describe('DottedMenuComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(MovieCardMenuComponent);
     component = fixture.componentInstance;
+    component.movie = new Movie(1, "testTitle", "testTagline", 5.6, 1000,
+      "2020-06-06", "https://image.tmdb.org/t/p/w500/3kcEGnYBHDeqmdYf8ZRbKdfmlUy.jpg", "testOverview", 6000, 10000, ["g1", "g2"],
+      120);
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+
+  it('change() test', () => {
+    component.change();
+    expect(component.isMark).toBe(true);
+    expect(component.headerMenu.nativeElement.style.display).toBe('block');
+    expect(component.header.nativeElement.style.justifyContent).toBe('flex-end');
+  });
+  it('mouseOut() test', () => {
+    component.mouseOut();
+    component.isMark = false;
```

Comment:
it looks as a bad component design if you only want to test `.mouseOut()` and changing manually an inner state along with it. Can you refactor the component to address that?